.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "developers.home-assistant",
+    "dockerFile": "Dockerfile",
+    "appPort": [
+        3000
+    ],
+    "postCreateCommand": "yarn install",
+    "extensions": [
+        "davidanson.vscode-markdownlint",
+        "eamodio.gitlens",
+        "editorconfig.editorconfig",
+        "github.vscode-pull-request-github",
+        "mrmlnc.vscode-scss",
```

Comment:
This project does not use scss?

Reply:
It could but we don't have it enabled (will drop)